src/api/java/com/minecolonies/api/colony/requestsystem/requestable/Stack.java

```diff
@@ -28,50 +30,58 @@
     /**
      * If meta should match.
      */
-    private boolean matchMeta = false;
+    private boolean matchMeta;
 
     /**
      * If NBT should match.
      */
-    private boolean matchNBT = false;
+    private boolean matchNBT;
 
     /**
      * If oredict should match.
      */
-    private boolean matchOreDic = false;
+    private boolean matchOreDic;
 
     /**
      * The required count.
      */
-    private int count = 0;
+    private int count;
+
+    /**
+     * The required count.
+     */
+    private int minCount;
 
     @NotNull
-    private ItemStack result = ItemStackUtils.EMPTY;
+    private ItemStack result;
```

Comment:
Please add the Empty initialization back. It just prevents errors and is just a safeguard no reason to remove it.

Reply:
I fixed up the logic, now all constructors always initialize it correctly such that I'm getting an intellij warning too.